deployment/oss-vdb-test/k8s/gke/alpine-cve-convert/alpine-cve-convert.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: alpine-cve-convert
+spec:
+  schedule: "0 */1 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: 3600
+      template:
+        spec:
+          containers:
+          - name: alpine-cve-convert
+            image: gcr.io/oss-vdb/alpine-cve-convert
```

Comment:
I believe this has to be `gcr.io/oss-vdb-test/alpine-cve-convert` for `gke-deploy` to correctly substitute the image.